Make debug mode in SVGTextLayoutEngine functional again

nikolaszimmermann opened the pull request:
#### 459f81b38481aff16323a2fa88756fe2cbd28abe
<pre>
Make debug mode in SVGTextLayoutEngine functional again
<a href="https://bugs.webkit.org/show_bug.cgi?id=257155">https://bugs.webkit.org/show_bug.cgi?id=257155</a>

Reviewed by Rob Buis.

Make debug mode &apos;DUMP_TEXT_FRAGMENTS&apos; &gt; 0, work again.
It wasn&apos;t adapted to the changes in the surrounding code since
a few years, since the code isn&apos;t compiled, usually.

Rename to DUMP_SVG_TEXT_LAYOUT_FRAGMENTS for clarity.
No change in functionality, thus no new tests.

* Source/WebCore/rendering/svg/SVGTextLayoutEngine.cpp:
(WebCore::dumpTextBoxes): Fix build issues.

Canonical link: <a href="https://commits.webkit.org/264625@main">https://commits.webkit.org/264625@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/35748119669a906cd9c7658c51c07b813b5d55e2

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/8073 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/8359 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/8572 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/9726 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/8172 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/8078 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/10344 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/8269 "Built successfully") | [⏳ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/WPE-WK2-Tests-EWS "Waiting to run tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/8217 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/9297 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/7355 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/9847 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/6598 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/7386 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/14964 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/7721 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/7511 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/10869 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/7984 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/6496 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/7295 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/1967 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/11504 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/7725 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rwlbuis commented on Source/WebCore/rendering/svg/SVGTextLayoutEngine.cpp:
```diff
@@ -249,23 +249,23 @@ void SVGTextLayoutEngine::layoutInlineTextBox(SVGInlineTextBox& textBox)
 #if DUMP_TEXT_FRAGMENTS > 0
```
This seems a very generic name, maybe we can consider making the name clear so it deals with SVG text engine.

nikolaszimmermann replied:
Ok.

nikolaszimmermann replied:
DUMP_SVG_TEXT_LAYOUT_FRAGMENTS?

rwlbuis replied:
Yep, that is clear enough.

rwlbuis reviewed (commented):
LGTM with suggestion/idea.

rwlbuis reviewed (approved):
LGTM.

nikolaszimmermann committed:
Make debug mode in SVGTextLayoutEngine functional again
https://bugs.webkit.org/show_bug.cgi?id=257155

Reviewed by Rob Buis.

Make debug mode 'DUMP_TEXT_FRAGMENTS' > 0, work again.
It wasn't adapted to the changes in the surrounding code since
a few years, since the code isn't compiled, usually.

Rename to DUMP_SVG_TEXT_LAYOUT_FRAGMENTS for clarity.
No change in functionality, thus no new tests.

* Source/WebCore/rendering/svg/SVGTextLayoutEngine.cpp:
(WebCore::dumpTextBoxes): Fix build issues.

Canonical link: https://commits.webkit.org/264625@main
